pages.id/common/chmod.md

```diff
@@ -0,0 +1,36 @@
+# chmod
+
+> Mengubah hak akses pengguna ([u]ser) suatu file atau direktori.
```

Comment:
How did the `[u]ser` get in there? This isn't in the original page.

Reply:
```suggestion
> Mengubah hak akses pengguna suatu file atau direktori.
```